gateway/service/proof/ProofService.ts

```diff
@@ -3,6 +3,18 @@ import { asL2Provider, CrossChainMessenger, L2Provider } from "@eth-optimism/sdk
 import { BigNumber, ethers } from "ethers";
 import { keccak256 } from "ethers/lib/utils";
 import { CreateProofResult, EthGetProofResponse, StorageProof } from "./types";
+
+
+export enum StorageLayout {
+    /**
+     * address,uint,bytes32,bool
+     */
+    NORMAL,
```

Comment:
To be consistent with solidity. `NORMAL` could be renamed to `FIXED` https://docs.soliditylang.org/en/v0.8.20/types.html#fixed-size-byte-arrays